CHANGES.md

```diff
@@ -5,6 +5,30 @@ for a full listing of issues for each release.
 
 --------------------------------------------------------------------------------
 
+## isodatetime 2.0.1 (Released 2019-07-23)
```

Comment:
No bug fixes, but some updated requirements... should it be 2.1.0 instead?

Reply:
I am torn. You are probably right as we are changing the namespace etc. However, we don't really have real known users for this version of the library yet, so either way is probably OK.

Reply:
Good point :+1: